api_core/tests/unit/test_page_iterator.py

```diff
@@ -69,6 +70,18 @@ def test_iterator_calls_parent_item_to_value(self):
         item_to_value.assert_called_with(parent, 12)
         assert page.remaining == 97
 
+    def test_raw_page(self):
+        parent = mock.sentinel.parent
+        item_to_value = mock.sentinel.item_to_value
+
+        raw_page = mock.sentinel.raw_page
+
+        page = page_iterator.Page(parent, (1, 2, 3), item_to_value, raw_page=raw_page)
+        assert page.raw_page is raw_page
+
+        with self.assertRaises(AttributeError):
```

Comment:
`pytest.raises`? https://docs.pytest.org/en/latest/getting-started.html#assert-that-a-certain-exception-is-raised
```suggestion
        with pytest.raises(AttributeError):
```

You can run the tests locally by installing nox.

```pip install nox```

and run the unit test session from the `api_core/` directory.

```nox -s unit```

This will handle setting up the virtualenv and installing dependencies.

Reply:
Thanks! I was previously using `python -m nox -r sessions unit-3.6`, which was choking on import and environment.

Reply:
Fixed, too.